mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.kt

```diff
@@ -240,7 +244,9 @@ class WidgetListFragment : Fragment(), WidgetAdapter.ItemClickListener {
                 }
             }
             if (ShortcutManagerCompat.isRequestPinShortcutSupported(context)) {
-                menu.add(Menu.NONE, CONTEXT_MENU_ID_PIN_HOME, Menu.NONE, R.string.home_shortcut_pin_to_home)
+                menu.add(Menu.NONE, CONTEXT_MENU_ID_PIN_HOME_WHITE, Menu.NONE,
+                    R.string.home_shortcut_pin_to_home_bright)
+                menu.add(Menu.NONE, CONTEXT_MENU_ID_PIN_HOME_BLACK, Menu.NONE, R.string.home_shortcut_pin_to_home_dark)
```

Comment:
Having the color options just besides each other seems somewhat random menu design to me. How about either
- making this a preference or
- using a submenu?

Reply:
> using a submenu?

:+1: 

I added a submenu, but now I cannot get the menuInfo of the submenu entries.

Reply:
I found a solution.